go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/observatorium/thanos-receive-controller
 
-go 1.14
+go 1.16
```

Comment:
Ditto. We don't need to change the min required version, this is for that. We don't use any features specific to 1.16 and above. This could stay the same, we shouldn't force it on the users. We can update the versions in the images.

Reply:
thanks @kakkoyun , I have a question, if we do not modify go.mod to use go 1.16.  you mean we need to update Dockerfile to use go 1.16?  like this:

```
FROM golang:1.16-alpine3.12 as builder

WORKDIR /workspace

COPY . .

RUN apk update && apk upgrade && apk add --no-cache alpine-sdk

RUN  make thanos-receive-controller

FROM gcr.io/distroless/static:latest

COPY --from=builder /workspace/thanos-receive-controller /usr/bin/thanos-receive-controller

USER 65534

ENTRYPOINT ["/usr/bin/thanos-receive-controller"]

```

Reply:
Yes, exactly

Reply:
@squat thanks. I have updated Dockerfile. pls help review.